rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -21,7 +21,7 @@
                 <javaclass references="org.hibernate.search.{*}" />
             </when>
             <perform>
-                <classification title="Hibernate Search API usage" effort="0" category-id="mandatory">
+                <classification title="Hibernate Search - API usage" effort="0" category-id="optional">
```

Comment:
ok, classifications are perhaps by its usage all Optional. No problem with this.